python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -396,7 +396,7 @@ def getMjd(self, dataId):
         elif not dateObs.endswith("Z"):
             dateObs += "Z"
 
-        return dafBase.DateTime(dateObs).get(dafBase.DateTime.MJD)
+        return dafBase.DateTime(dateObs, timeSystem).get(dafBase.DateTime.MJD)
```

Comment:
The time scale for `get` defaults to TAI. Thus with the old behavior the ISO string would be treated as UTC, but the returned MJD would be TAI, introducing a 30-ish second offset. But that is not documented as the desired behavior. Please either document it or fix it by providing the time system argument to `get` so the method is self-consistent (UTC in, UTC out, or TAI in, TAI out).